### DISCUSSION TITLE
Light experience & recommendations sought

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#949] [Q&A]
Our pool builder installed Pentair GloBrite lights in our pool.  To say they are ridiculously priced pieces of junk would be too kind.  Two lights failed within 2 years and one has failed for the second time.  Of course, they're now out of warranty...

So, the question is, where do I  go from here?  I believe MircoBrite lights can fit in the niches, but it's not clear from what I've seen that they're any better than GloBrites.  If I  get MicroBrites, do I need to replace all of the pool lights or will the light output from the MircoBrites look "about"  the same as the GloBrites?  Is there a better option than Pentair's that will fit in the niches?

Thanks for sharing your experiences & recommendations!!!

**Comment by NWSAlpine** [comment]
I had 4 globrites fail over the years from a 2018 build. I initially replaced 2 with microbrites but I'm not happy. They are much dimmer than the globrites (200 lumen difference). 

There are some reports the Jandy 1.5" nicheless lights fit the globrite niche but don't pully secure. Not that big of a deal for this style of light. https://www.troublefreepool.com/threads/pentair-globrite-replacement-option.287769/ 

Pentair redesigned the Globrite with a new led board and new body as of 12/23. This is to resolve the cracking issue. I just bought 2 of the new lights and installed recently. Time will tell if they really fixed them or not. I moved my microbrites to the bubblers where the lumen difference isn't as big of a deal. The microbrite colors are slightly different than the globrites so don't mix and match in the same area you are lighting or it will look weird.

**Reply by FlaMike** [reply]
Thank you so much for your reply!  It sounds like the Micro Brites are not a worthy improvement over the Glo Brites.  Given their cost, I am tired of buying junk from Pentair.  And I have no interest in trying the "new and improved" GloBrites.  Let them find some other sucker to drop thousands on their less than robust lights!

I may go the Jandy route (P version so Nixie can easily control them as if they were Pentair lights).  Of course, that path means I need to find a pool contractor who will either sell me the lights or sell & install them.  And I can only hope they're an improvement--5 or 10 years of life for a 50,000 MTBF product?  We should be so lucky!!!

Sooo tired of over paying for junk pool equipment!

Any other thoughts and suggestions are most welcome and appreciated!!!  Thanks again for your reply.
